.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+Metrics/LineLength:
+  Enabled: false
```

Comment:
If this is a real bummer, I'd recommend setting it to something slightly higher than reasonable like 120. We really shouldn't have code over 120 characters wide.